modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -388,6 +388,10 @@ function getComments($issueID)
 
     //looping by reference so can edit in place
     foreach ($unformattedComments as &$comment) {
+        $comment["dateAddedFormatted"] = date(
```

Comment:
And I most certainly did not want to write my own date-formatting routines. I wasn't about to reinvent the wheel. Or maybe I should write the date formatter as an Ajax call that uses PHP's own date() function? =P